tests/Application.Tests/ExecutorTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Symbolica.Application
+{
+    public class ExecutorTests
+    {
+        [Theory]
+        [ClassData(typeof(FailTestData))]
+        private async Task ShouldFail(string directory,
+            bool useSymbolicGarbage, bool useSymbolicAddresses, bool useSymbolicContinuations)
+        {
+            var result = await Executor.Run(directory,
+                useSymbolicGarbage, useSymbolicAddresses, useSymbolicContinuations);
+
+            result.IsSuccess.Should().BeFalse();
```

Comment:
Map the result's error and then compare the whole structure to `Result.Failure(await File.ReadAllLinesAsync(Path.Combine(directory, "symbols")))`

Reply:
Nah